contracts/cw721-expiration/src/contract_tests.rs

```diff
@@ -41,41 +70,141 @@ fn setup_contract(deps: DepsMut<'_>, expiration_days: u16) -> Cw721ExpirationCon
 #[test]
 fn proper_instantiation() {
     let mut deps = mock_dependencies();
-    let contract = Cw721ExpirationContract::default();
+    let contract = Cw721ExpirationContract::<
+        DefaultOptionalNftExtension,
+        DefaultOptionalNftExtensionMsg,
+        DefaultOptionalCollectionExtension,
+        DefaultOptionalCollectionExtensionMsg,
+        Empty,
+        Empty,
+        Empty,
+    >::default();
```

Comment:
Is there a way we can improve the developer experience around this? Especially for tests?

Reply:
Defined an opionated `DefaultCw721ExpirationContract` (aside from generic `Cw721ExpirationContract`) for ease-of-use. Looks now like this:

```rust
let contract = DefaultCw721ExpirationContract::default();
```